README.md

```diff
@@ -151,22 +151,34 @@ The build script does the following:
 
 The buildroot uses ~4 GiB of disk space and can take almost an hour to build.
 
-### OpenDingux
+For OpenDingux builds `mksquashfs` needs to be installed.
 
-The OpenDingux build uses the buildroot at `$HOME/buildroot-2018.02.9-opendingux-musl`.
+### RetroFW (RS97, RG300, LDK)
+
+The RetroFW build uses the buildroot at `$HOME/buildroot-2018.02.9-retrofw`.
 
 ~~~ bash
-Packaging/OpenDingux/build-opendingux-sdl1.sh
+Packaging/OpenDingux/build-retrofw.sh

```

Comment:
Maybe a better folder/title name would be RetroGamerConsols/Retro Gamer Consols as that appears to be the common denominator for all of these devices.

Reply:
I'd like to not rename it yet. I have a number of commits in the controller supports branch on top of it and it'd be nice to minimize merge conflicts when we eventually add controller support to mainline.

Reply:
Besides, RetroFW is a fork of OpenDingux

Reply:
Sounds good thanks for the clarification